scripts/lint-style.py

```diff
@@ -340,6 +350,8 @@ def format_errors(errors):
             output_message(path, line_nr, "ERR_AUT", "Authors line should look like: 'Authors: Jean Dupont, Иван Иванович Иванов'")
         if errno == ERR_TAC:
             output_message(path, line_nr, "ERR_TAC", "Files in mathlib cannot import the whole tactic folder, nor tactic.omega or tactic.observe")
+        if errno == ERR_UNF:
+            output_message(path, line_nr, "ERR_UNF", "Use of unfreeze_local_instances is discouraged and leads to performance problems")
```

Comment:
Do you think you can squeeze in the "solution/alternatives"? We want the error message to be short, but mentioning `unfreezingI` would be good